src/www/system_usermanager.php

```diff
@@ -434,6 +434,19 @@ function get_user_privdesc(& $user)
         $('#otp_qrcode').show();
```

Comment:
cluster the sections together as in bundle the "unhide" action into this method??

Reply:
yes, so you don't have to manipulate the existing input / add the textarea